e2e/test/scaling/provider.go

```diff
@@ -94,30 +113,25 @@ func (p *Provider) NumMasters() (int, error) {
 }
 
 func (p *Provider) NumWorkers() (int, error) {
-	customObject, err := p.hostFramework.G8sClient().ProviderV1alpha1().AzureConfigs("default").Get(p.clusterID, metav1.GetOptions{})
+	ctx := context.TODO()
```

Comment:
```suggestion
	ctx := context.Background()
```
https://github.com/giantswarm/fmt/blob/master/go/context.md#guidelines